src/datepicker/datepicker.ts

```diff
@@ -47,36 +47,63 @@ export interface NgbDatepickerNavigateEvent {
 @Component({
   exportAs: 'ngbDatepicker',
   selector: 'ngb-datepicker',
-  host: {'class': 'd-inline-block'},
   styles: [`
-    .month:first-child {
-      padding-left: 0 !important;
+    :host {
+      display: inline-block;
+      border: 1px solid rgba(0, 0, 0, 0.125);
+      border-radius: 0.25rem;
+    }
+    .ngb-dp-navigation {
+      padding-top: 0.25rem;
+    }
+    .ngb-dp-months {
+      padding-bottom: 0.25rem;
+    }
+    .ngb-dp-month, .ngb-dp-month-spacer {
+      display: inline-block;
+    }
+    .ngb-dp-month-name {
+      font-size: larger;
+      height: 2rem;
+      line-height: 2rem;
+    }    
+    .ngb-dp-month-spacer {
+      width: 1rem;      
+      height: 4rem;
+      border-bottom: 1px solid rgba(0, 0, 0, 0.125);    
+    }
+    .ngb-dp-month-spacer-sm {
+      width: 0.25rem;
+      border-bottom: 1px solid rgba(0, 0, 0, 0.125);    
     }
   `],
   template: `
     <template #dt let-date="date" let-currentMonth="currentMonth" let-selected="selected" let-disabled="disabled">
        <div ngbDatepickerDayView [date]="date" [currentMonth]="currentMonth" [selected]="selected" [disabled]="disabled"></div>
     </template>
-
-    <ngb-datepicker-navigation *ngIf="navigation !== 'none'"
-      [date]="months[0]?.firstDate"
-      [minDate]="_minDate"
-      [maxDate]="_maxDate"
-      [disabled]="disabled"
-      [showWeekNumbers]="showWeekNumbers"
-      [showSelect]="navigation === 'select'"
-      (navigate)="onNavigateEvent($event)"
-      (select)="onNavigateDateSelect($event)">
-    </ngb-datepicker-navigation>
-
-    <table>
-      <tr *ngIf="navigation !== 'select' || displayMonths > 1">
-        <td *ngFor="let month of months" class="text-center font-weight-bold">
-          {{ i18n.getMonthName(month.number) }} {{ month.year }}
-        </td>
-      </tr>
-      <tr>
-        <td *ngFor="let month of months" class="pl-1 month">
+    
+    <div class="ngb-dp-navigation bg-faded">
+      <ngb-datepicker-navigation *ngIf="navigation !== 'none'"
+        [date]="months[0]?.firstDate"
+        [minDate]="_minDate"
```

Comment:
Should private variables be used in templates? Not sure if these should be public or not. Maybe they are, just going by the name of the var here.